apps/system/utils/utils_proc.c

```diff
@@ -93,7 +93,7 @@ int utils_proc_pid_foreach(procentry_handler_t handler)
 	return ret;
 }
 
-int utils_readfile(FAR const char *filepath, char *buf, int buflen, utils_print_t print_func)
+int utils_readfile(FAR const char *filepath, char *buf, int buflen, void *arg, utils_handler_t handler)
```

Comment:
How about changing a sequence of handler and args for better understanding?
That's because args is argument passed to handler, not used in this function.